model/project_matrix.go

```diff
@@ -388,12 +388,12 @@ func buildMatrixVariant(axes []matrixAxis, mv matrixValue, m *matrix, ase *axisS
 		}
 		matchers, errs := r.If.evaluatedCopies(ase) // we could cache this
 		if len(errs) > 0 {
-			return nil, errors.Errorf("evaluating rules for matrix %v: %v", m.Id, errs)
+			return nil, errors.Errorf("evaluating rules for matrix %s: %v", m.Id, errs)
```

Comment:
We can't use `errors.Wrapf` since `errs` is an error slice. I didn't feel like refactoring the matrix code to use catchers since matrix definitions for tasks are supposed to be deprecated.